File: templates/Uwp/_composition/MVVMLight/Feature.ShareTargetVB/ViewModels/ViewModelLocator_postaction.vb
Namespace ViewModels
    Public Class ViewModelLocator
        Private Sub New()
            '{[{
            If SimpleIoc.[Default].IsRegistered(Of NavigationServiceEx)() Then
                Return
            End If
            '}]}
            SimpleIoc.[Default].Register(Function() New NavigationServiceEx())
            '^^
            '{[{
            Register(Of wts.ItemNameViewModel, wts.ItemNamePage)()
                '}]}
        End Sub

        '{[{

        Public ReadOnly Property wts.ItemNameViewModel As wts.ItemNameViewModel
            Get
                Return SimpleIoc.[Default].GetInstance(Of wts.ItemNameViewModel)()
            End Get
        End Property
        '}]}
    End Class
End Namespace

File: templates/Wpf/_comp/MVVMLight/Page.AddViewModel/ViewModels/ViewModelLocator_postaction.cs
public class ViewModelLocator
{
//^^
//{[{
    public wts.ItemNameViewModel wts.ItemNameViewModel
        => SimpleIoc.Default.GetInstance<wts.ItemNameViewModel>();
//}]}

    public ViewModelLocator()
    {
//^^
//{[{
        Register<wts.ItemNameViewModel, wts.ItemNamePage>();
//}]}
    }
}
